pymeasure/instruments/lecroy/lecroyT3DSO1204.py

```diff
@@ -33,7 +36,10 @@
 log = logging.getLogger(__name__)
 log.addHandler(logging.NullHandler())
 
-_MATCH_FLOAT = re.compile(r'-? *[0-9]+\.?[0-9]*(?:[Ee] *-? *[0-9]+)?')
+
+def _ceildiv(a, b):
```

Comment:
`math.ceil(a//b)` does not do what you want?

Reply:
Yes it is true. Applied in commit 2b92a7c1795e42dd8777551329450a2ff8ac6c08

Reply:
The waveform download is not working with `math.ceil(a//b)`. I guess that the output of `a//b` is already floor-ed. I reverted this change in commit 28a2145c817f6bc96f864e60d875c7882de48a71